### PR TITLE
Update build.sh

### PR DESCRIPTION
**Pull request by DanWBR**
- Added automatic install of required packages
- Updated static paths to use the current script directory as the base one for building the source and setting up the rpi sysroot

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+
+export BASE_DIR=$(dirname "$SCRIPT")
 
 export BUILD_DIR=$BASE_DIR/build
 export RPI_ROOT=$BASE_DIR/rpi
 
+# install required dependencies
+
+sudo apt-get install apt qemu-user-static debootstrap clang build-essential g++-arm-linux-gnueabihf libglib2.0-dev 
+
```
Comment (mterwoord) [inline build.sh]: I don't like these dependencies in the script. Could you remove them?

**Review by DanWBR — COMMENTED** [review]
Removed command to install dependencies

**Comment by mterwoord** [comment]
Thanks!

**Comment by DanWBR** [comment]
You're welcome!

**Pull request merged**